src/Widgets/OverlookWidget.php

```diff
@@ -86,7 +92,7 @@ public function getData(): array
                     'name' => $title,
                     'raw_count' => $this->formatRawcount($rawCount),
                     'count' => $this->convertCount($rawCount),
-                    'icon' => $res->getNavigationIcon(),
+                    'icon' => $customIcon ?? $res->getNavigationIcon(),
```

Comment:
This needs to use the `?:` operator since array_search() returns `false` if it doesn't find a match and not `null`.

```php
'icon' => $customIcon ?: $res->getNavigationIcon(),
```